src/Components/Home.tsx

```diff
@@ -18,14 +30,123 @@ export default function Home({ user, setUser }: Props) {
     navigate("/login");
   };
 
+  // Mockup *Temporário*
+  const editais = {
```

Comment:
Sugestão: Criar mock de acordo com a chave `user`. Se tiver `user`, fica como tá. Se `user` for `falsy`, fica sem a chave `inscritos`.